Update ingestion.mdx (#663)
stellar/new-docs: content/docs/run-api-server/ingestion.mdx

The following instructions assume the reingestion is done on AWS. However, they should be applicable to any other environment with equivalent capacity. In the same way, the instructions can be adapted to reingest only specific parts of the history.

### Prerequisites

> Before we begin, we make some assumptions around the environment required. Please refer to the [Prerequisites](./prerequisites.mdx) section for the current HW requirements to run Horizon reingestion (for either historical catch up) or real-time ingestion (for staying in synch with the ledger). A few things to keep in mind:

Reviewer: mix-up of parentheticals here, I think
```suggestion
Before we begin, we make some assumptions around the environment required. Please refer to the [Prerequisites](./prerequisites.mdx) section for the current HW requirements to run Horizon reingestion for either historical catch up or real-time ingestion (for staying in sync with the ledger). A few things to keep in mind:
```

>   1. For reingestion, the more parallel workers are provisioned to speed up the process, the larger the machine size is required in terms of RAM, CPU, IOPS and disk size. The size of the RAM per worker also increases over time (14GB RAM / worker as of mid 2022) due to the growth of the ledger. HW spec can be downsized once Re-ingestion is completed. 

Reviewer: nit: consistency 

```suggestion
  1. For reingestion, the more parallel workers are provisioned to speed up the process, the larger the machine size is required in terms of RAM, CPU, IOPS and disk size. The size of the RAM per worker also increases over time (14GB RAM / worker as of mid 2022) due to the growth of the ledger. HW specs can be downsized once reingestion is completed. 
```

  2. [Horizon](./installing.mdx) version 2.1x or newer installed on the machine from (1).

>   3. [Core](https://github.com/stellar/stellar-core) version 18.5 or newer  installed on the machine from (1).

Reviewer: Should we just stop mentioning a version here and just say "the latest version of stellar-core" and link to https://github.com/stellar/stellar-core/releases/latest?

Reviewer: And likewise for Horizon?

Author: Yes good idea!

>   4. A Horizon database where to reingest the history. Preferably, the database should be empty to minimize storage (Postgres accumulates data during usage, which is only deleted when `VACUUM`ed) and have the minimum spec's for reingestion as outlined in [Prerequisites](./prerequisites.mdx).

Reviewer: nit: `specs` isn't possessive 

```suggestion
  4. A Horizon database where to reingest the history. Preferably, the database should be empty to minimize storage (Postgres accumulates data during usage, which is only deleted when `VACUUM`ed) and have the minimum specs for reingestion as outlined in [Prerequisites](./prerequisites.mdx).
```
